rc/windowing/kitty.kak

```diff
@@ -15,7 +15,11 @@ kitty-terminal <program> [<arguments>]: create a new terminal as a kitty window
 The program passed as argument will be executed in the new terminal' \
 %{
     nop %sh{
-        kitty @ new-window --no-response --window-type $kak_opt_kitty_window_type --cwd "$PWD" "$@"
+        if [[ -z ${kak_client_env_KITTY_LISTEN_ON:-} ]]; then
```

Comment:
`[[` is a bashism, for portability just use `[`. Also, trying to "default" (`:-`) to an empty string is redundant, because an unset environment variable already expands to an empty string.

```suggestion
        if [ -z "$kak_client_env_KITTY_LISTEN_ON" ]; then
```